students/gene/project/node_app.py

```diff
@@ -0,0 +1,89 @@
+
+
+from node_db import Node, NodeStorage
+
+
+def main_menu_usage():
+    print ("\n\n"
+           "------------------------------------------------------\n"
+           "|               Divice Database                        |\n"
+           "------------------------------------------------------"
+           "\n\nSelect from one of these options:\n\n"
+           "(1) Add Device\n"
+           "(2) Remove Device\n"
+           "(3) Create device inventory report\n"
+           "(4) Quit\n")
+
+
+def main_menu():
+
+    while True:
+        main_menu_usage()
+        try:
+            answer = int(input("Option: "))
+            print("\nOption selected: {}\n\n".format(answer))
+        except ValueError:
+            answer = 0
+            print("\n\n****ERROR: Please choose a numeric option****\n\n")
+            continue
+
+        if answer == 4:
+            print("\nQuiting.....")
+            break
+
+        elif answer == 1:
+            node_id = input("Device name: ").strip().upper()
```

Comment:
this is a lot of repeated code -- can you use setattr() and a loop to make it less repeated?

and/or store it all ina dict, then create the Node object from a dict?

IT would look somethign like:
```
attributes = ["site_id", "console_id', ...]
for at in attributes:
    atts[at] = input(f"please type in: {at}:").strip()

```